easy_music_generator/tests/test_easy_music_generator.py

```diff
@@ -0,0 +1,29 @@
+import os
+import glob
+import unittest
+import easy_music_generator as emg
+
+
+class TestEasyMusicGenerator(unittest.TestCase):
+    """
+    Unit tests for easy_music_generator.py
+    """
+
+    def setUp(self):
```

Comment:
Add docstring
'''
This method prepares the setup for the test by removing pre-existing midi files from the output folder. 
'''